lib/trento/infrastructure/operations/amqp/processor.ex

```diff
@@ -75,7 +77,7 @@ defmodule Trento.Infrastructure.Operations.AMQP.Processor do
   end
 
   defp maybe_request_discovery(operation, :UPDATED, group_id)
-       when operation in [:saptune_solution_apply, :saptune_solution_change] do
+       when operation in HostOperations.values() do
```

Comment:
I think we need to be more picky on this one, and leave it as it was.
Here we just want launch a saptune discovery, and most probably not all host operations need this

Reply:
0b6e4bd35f1ff67eb3a01e0f436f87e89dbf4bd8